content/docs/getting-started/fluvio-cli.md

```diff
@@ -10,35 +10,64 @@ up, managing, and interacting with Fluvio.
 
 -> Please note that we currently only support Linux and MacOS
 
-#### Download
+#### Download and Install
 
-Download the Fluvio CLI from our [github releases] page
-
-[github releases]: https://github.com/infinyon/fluvio/releases
-
-You'll want to rename the binary to `fluvio` and mark it as executable. For the
-following command, make sure to use the filename of the release you downloaded.
+Downloading and installing Fluvio is as simple as running the following command!
 
 ```bash
-$ mv fluvio-v0.6.0-alpha.2-x86_64-apple-darwin fluvio
-$ chmod +x ./fluvio
+$ curl -fsS https://packages.fluvio.io/v1/install.sh | bash
 ```
 
-Then, install the CLI by moving it into a bin folder in your PATH, like this
+If everything works successfully, you should see output similar to this:
 
 ```bash
-$ sudo mv ./fluvio /usr/local/bin/
+fluvio: ⏳ Downloading Fluvio 0.6.0-alpha.5 for x86_64-apple-darwin...
+fluvio: ⬇️ Downloaded Fluvio, installing...
+fluvio: ✅ Successfully installed ~/.fluvio/bin/fluvio
+fluvio: ☁️ Installing Fluvio Cloud...
```

Comment:
can you also change this icon with the one I suggested in slack?

Reply:
That requires code changes to the actual CLI which will take time since we have to go through a cycle of publishing it before it will take effect. If I update the doc before that happens then it won't be an accurate display of what the CLI does. I will kick off those changes in the CLI and when they land I will update it here

Reply:
I didn't realize that the CLI displays the icon fine. In that case we should leave it alone.